src/app/pages/psbt-request/components/psbt-decoded-request/psbt-decoded-request-nodes/inputs/psbt-placeholder-inputs.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, Flex, Text, color } from '@stacks/ui';
+
+import { BtcIcon } from '@app/components/icons/btc-icon';
+import { Flag } from '@app/components/layout/flag';
+import { SpaceBetween } from '@app/components/layout/space-between';
+
+export function PsbtPlaceholderInputs() {
+  return (
+    <Box background="white" borderTopLeftRadius="16px" borderTopRightRadius="16px" p="loose">
+      <Text fontWeight={500}>Inputs</Text>
+      <Flag align="middle" img={<BtcIcon />} mt="loose" spacing="base">
```

Comment:
V similar layout to elsewhere, could create a new component

Reply:
Agree, some of these got redundant while building. I'll simplify them. 👍